### PR TITLE
SiriusPVName: fix strip that was returning a str object.

### PR DESCRIPTION
**Pull request by xresende**

### CODE REVIEW

File: siriuspy/siriuspy/namesys/implementation.py [hunk 1]
```diff
@@ -255,6 +258,14 @@ def __ge__(self, other):
         """Greater-or-equal operator."""
         return self.__gt__(other) or self.__eq__(other)
 
+    def strip(self, *args, **kwargs):
+        """."""
+        return SiriusPVName(super().strip(*args, **kwargs))
+
+    def replace(self, *args, **kwargs):
```
[the remark this answers is not in the document]
Comment (xresende) [reply siriuspy/siriuspy/namesys/implementation.py]: 1) I had to subtitute _-SP_ for  _Ref-Mon_, both for _Current-SP_ and _SOFBCurrent-SP_ power supply properties. It is very awkward to do this without `str.replace`
2) I think is is a good idea to implement all `str` methods along the way. For exampple, class _PVFake_ does use `strip` in its implementation. If we don't want to go patching externally used packages that operate on strings, we should complete _SiriusPVName_.

**Comment by xresende** [comment]
> Comment in text. Just a reminder... wink

@fernandohds564 , I did not understand. Did I leave a comment print in the code?

File: siriuspy/siriuspy/namesys/implementation.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (fernandohds564) [reply siriuspy/siriuspy/namesys/implementation.py]: There is a method in `SiriusPVName` called `substitute`. This method is useful to change parts of the PV name which have a meaning, for example (`dev`, `sub`, `sec`, `dis`, `propty`, `propty_suffix`, `propty_name`, etc), and its use preferable, in my opinion, to the replace method whenever possible.
In the specific example you gave in (1), where you had to change from `-SP` to `Ref-Mon` it is indeed better to use `replace`. My comment was only to remember the existence of this method and how much clearer its use is in terms of code readability.

[the remark this answers is not in the document]
Comment (xresende) [reply siriuspy/siriuspy/namesys/implementation.py]: I know of `SiriusPVName.substitute`, @fernandohds564. It is indeed very useful. I used it in the code snipt I wrote to you guys at the gitter room about `simul` subpackage. Thanks.
I agree `substitute` is preferable whenever possible, although sometimes base-class methods might be judiciously used for efficiency.

**Pull request merged**